docs/guide/adapters/deconz.md

```diff
@@ -37,7 +30,14 @@ Add the correct baudrate to the `configuration.yaml` into the serial section.
 - For RaspBee2 it is 38400
 - For ConBee3 it is 115200
 
+<!--
+NOTE(manup) 2025-07-20: Following warning is **very** problematic advice as the old firmware doesn't handle some newer devices well and has various bugs which where ironed out by later versions.
+The 0x26780700 stable version is highly recommended to work solid with the new adapter rewrite.
+
+Reference: https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Firmware-Changelog
+
 **Warning:** Conbee 2 firmware versions newer than 0x26580700 will result in an unstable network with devices dropping randomly, see [Issue 9554](https://github.com/Koenkk/zigbee2mqtt/issues/9554)
```

Comment:
I think this can be removed? (maybe add a line here that `0x26780700` is recommended)